first_validation.py

```diff
@@ -54,7 +59,19 @@ def check_hashes(hash_fname):
         # Calculate actual hash for given filename.
         # Check actual hash against expected hash
         # Return False if any of the hashes do not match.
-    return False
+    def get_file_hash(path_fname):
+        contents = path_fname.read_bytes()
+        hash_value = sha1(contents).hexdigest()
+        return hash_value
+
+    hash_text = hash_fname.read_text().splitlines()
+    for line in hash_text:
+        hash,_,file = line.split(" ")
```

Comment:
Or use the default behavior of splitting on one or more spaces:

```suggestion
        hash, file = line.split()
```

Reply:
thanks. yes it's simpler this way. it's a habit for me to write this way i suppose. i'll pay attention to it.